website/identifiers/views.py

```diff
@@ -17,6 +17,8 @@
 def node_identifiers_post(auth, node, **kwargs):
     """Create identifier pair for a node. Node must be a public registration.
     """
+    # TODO this functionality exists in APIv2. When front end is entirely using
+    # v2 for minting DOI's, remove this view.
```

Comment:
Initially started to consolidate v1 and v2 views, but because of the different error handling, this started getting out of scope.  v1 view will be removed at some point.